pkgs/development/python-modules/validphys2/default.nix

```diff
@@ -21,10 +27,19 @@ buildPythonPackage rec {
   '';
 
   propagatedBuildInputs = [
+    lhapdf
     nnpdf
+    prompt_toolkit
     reportengine
+    requests
+    seaborn
+    validobj
   ];
 
+  passthru.optional-dependencies = {
+    complete = [ pandoc ];
```

Comment:
This doesn't make sense unless `extras_require["complete"]` is specified in `setup.py`.

Reply:
Fair enough, removed